hammer/drc/klayout/__init__.py

```diff
@@ -0,0 +1,101 @@
+# Klayout DRC plugin for Hammer
+#
+# See LICENSE for licence details.
+
+import os
+from textwrap import dedent as dd
+from typing import List, Optional, Dict, Any
+
+from hammer.logging import HammerVLSILogging
+from hammer.utils import deepdict
+from hammer.vlsi import HammerToolStep
+from hammer.vlsi import HammerDRCTool, TCLTool
+
+class Klayout(HammerDRCTool, TCLTool):
+
+    #=========================================================================
+    # overrides from parent classes
+    #=========================================================================
+    @property
+    def steps(self) -> List[HammerToolStep]:
+        return self.make_steps_from_methods([
+            # no steps
```

Comment:
Hm, this would prevent users from adding hooks... is there any situation where one would override or add things to the deck? Or, even though it would be different from similar plugins, could `run_klayout` be the singular step and we could leave some results post-processing to `do_post_steps`?